modules/twitter.js

```diff
@@ -0,0 +1,49 @@
+var config = require('config');
+var Twit = require('twit');
+
+var T = new Twit({
+  consumer_key: config.get('twitter.consumerKey'),
+  consumer_secret: config.get('twitter.consumerSecret'),
+  access_token: config.get('twitter.accessToken'),
+  access_token_secret: config.get('twitter.accessTokenSecret'),
+  timeout_ms: 60*1000,  // optional HTTP request timeout to apply to all requests.
+})
+
+
+function trigger(tweetURL, api, message) {
+	var threadID = message.threadID;
+	var tweetID = splitAtStatus(tweetURL);
```

Comment:
Does it make sense to recreate the twitter object for every command? Since the tokens are static, we could just have it in the main.js